guides/common/modules/proc_overriding-ansible-variables.adoc

```diff
@@ -12,42 +15,56 @@ To ensure that the variable that you override follows the correct order of prece
+For more information, see xref:importing-Ansible-roles_{context}[]
 
 .Procedure
 
 . In the {ProjectWebUI}, navigate to *Configure* > *Variables*.
 . Select the Ansible variable that you want to override and manage with {Project}.
 . Navigate to the *Default Behavior* area, and select the *Override* check box.
-. From the *Parameter Type* select the value type for validation.
-For example, a *string* or *boolean* variable.
+. In the *Parameter Type* field, select the value type for validation such as *string* or *boolean*.
+The types *array* and *hash* have further options for handling upon a variable match.
+See *Prioritize Attribute Order* below.
 . In the *Default Value* field, enter the default value that you want to use if there is no match for the variable.
-. Optional: If you do not want to display the Ansible variable in plain text, select the *Hidden Values* check box to display the content of the variable as asterisks in the {ProjectWebUI}.
+. Optional: If you do not want to display the value of the variable as plain text in the {ProjectWebUI}, select the *Hidden Value* check box to display the content of the variable as asterisks.
+This is useful for sensitive values such as passwords or secret tokens.
+. Optional: Expand the *Optional Input Validator* area and specify conditions that will be used to validate concrete values of the variable:
+* Select *Required* if you want to enforce users to fill in this variable.
+* In the *Validator Type* field, select how the value will be validated:
+** `list` - The value will be validated against an enumeration of allowed values.
+** `regex` - The value will be validated against a regular expression pattern.
+. Optional: In the *Prioritize Attribute Order* area, specify the order of priority to match an override with a host by host attributes.
+Order at the top takes higher precedence.
+The first match wins.
++
+You can combine multiple attributes into a single matcher key using a comma as the AND operation.
+For example, the matcher key of `hostgroup, environment` would expect matchers such as `hostgroup = "web servers"` AND `environment = production`.
++
+If you use the parameter type *array* or *hash*, you can further set:
+
+** *Merge Overrides* - Merges members of the arrays/hashes instead of replacing the whole array or hash. If the hashes contain the same key, the value is overwritten by the value of the host.
+** *Merge Default* - Adds the default value to the array or hash.
+** *Avoid Duplicates* - Ensures that the values in the array or hash are unique.
+. Optional: Expand the *Specify Matchers* area and specify criteria for selecting hosts on which the variable overrides.
 . To save the override settings, click *Submit*.
 
 To use the Ansible variable, add the variable as a parameter to your host or host group, or add the variable as a global parameter.
 
-.For a Host Group:
+== Adding the Variable to a Host
```

Comment:
Please move to a separate file or keep the "fake" heading; but IHMO this looks like a procedure to me.